src/js/functions.js

```diff
@@ -741,6 +741,42 @@ if(typeof jQuery != 'undefined') {
   });
 }
 
+function loadRecentContent() {
+  var icons = {
+    'blog': 'bullhorn',
+    'tutorial': 'education'
+  };
+  console.log('loading recent content');
```

Comment:
I'd remove the console.logs. Not sure if it's true anymore, but they used to cause problems on IE. But maybe not an issue anymore.